<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrainingGenWidget</class>
 <widget class="QWidget" name="TrainingGenWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Texts per stage:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QComboBox" name="layoutComboBox"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Characters per lesson:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QSpinBox" name="lessonLengthSpinBox">
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Layout:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QPushButton" name="generateButton">
       <property name="text">
        <string>Generate</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="lessonsPerStageSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="value">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="7">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>This will generate a series of lessons to teach you a given keyboard layout.

 It will start with just the index fingers on the home row.
It will then add more fingers, and eventually expand to include the upper and lower row.

Go to the sources tab to find the generated lessons.</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
